provider/providerutil/providerutil.go

```diff
@@ -112,6 +113,74 @@ func AppiumAvailable() bool {
 	return true
 }
 
+// Check if the GADS Appium plugin is installed on NPM
+func CheckAppiumPluginInstalledNPM() bool {
+	cmd := exec.Command("npm", "list", "-g", "appium-gads", "--depth=0")
+	err := cmd.Run()
+	if err != nil {
+		return false
+	}
+	return true
+}
+
+// Get the version of the Appium plugin currently installed on NPM
+func GetAppiumPluginNPMVersion() (string, error) {
+	cmd := exec.Command("npm", "list", "-g", "appium-gads", "--depth=0")
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+	lines := strings.Split(string(out), "\n")
+	for _, line := range lines {
+		if strings.Contains(line, "appium-gads@") {
+			parts := strings.Split(line, "@")
+			if len(parts) == 2 {
+				return strings.TrimSpace(parts[1]), nil
+			}
+		}
+	}
+	return "", fmt.Errorf("Could not get GADS Appium plugin version on NPM")
+}
```

Comment:
Should it be removed?

Reply:
I've added this intentionally, probably on next version each version will have hardcoded supported plugin version so the provider can update automatically if needed. We do not use it now but we will probably use it in the future :)